scripts/maven-release-publisher.yml

```
@@ -0,0 +1,38 @@
+version: 0.2
+env:
+  shell: /bin/sh
+  secrets-manager:
+    ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME: awsmobilesdk/android/sonatype:username
+    ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD: awsmobilesdk/android/sonatype:password
+    ORG_GRADLE_PROJECT_signingPassword:  awsmobilesdk/android/signing:password
+    ORG_GRADLE_PROJECT_signingKeyId:  awsmobilesdk/android/signing:keyId
+    ORG_GRADLE_PROJECT_signingInMemoryKey: awsmobilesdk/android/signing:inMemoryKey
+    ORG_GRADLE_PROJECT_signingSecretKeyRingFile:  awsmobilesdk/android/signing:secretKeyRingFile
+phases:
+  install:
+    commands:
+      - echo 'Install phase starting'
+    finally:
+      - echo 'Install phase completed.'
+  pre_build:
+    commands:
+      - echo 'Pre-build phase starting'
+    finally:
+      - echo 'Pre-build phase completed.'
+  build:
+    commands:
+      - echo 'Build phase starting.'
+      - |
+        # List all available gradle tasks, grep for the uploadArchive tasks, and then use cut to strip the
+        # task description and just return the name of the task, one for each module (e.g. aws-api:uploadArchives)
+        for task_name in $(./gradlew tasks --all | grep uploadArchives | cut -d " " -f 1); do
+          echo "Gradle task $task_name"
+          JAVA_HOME=$JDK_8_HOME ./gradlew clean build $task_name;
+        done
```

Actually, `clean build` should be done for all modules, not inside the for loop.  Otherwise, it's going to cause dependency problems.

```suggestion
        JAVA_HOME=$JDK_8_HOME ./gradlew clean build
        for task_name in $(./gradlew tasks --all | grep uploadArchives | cut -d " " -f 1); do
          echo "Gradle task $task_name"
          JAVA_HOME=$JDK_8_HOME ./gradlew $task_name;
        done
```